recipes/qxlsx/all/conanfile.py

```diff
@@ -63,14 +72,18 @@ def build(self):
     def package(self):
         copy(self, "LICENSE", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
         cmake = CMake(self)
+        cmake.configure(build_script_folder="QXlsx")
         cmake.install()
         rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
 
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "QXlsx")
         self.cpp_info.set_property("cmake_target_name", "QXlsx::Core")
         # TODO: back to global scope in conan v2 once cmake_find_package* generators removed
-        self.cpp_info.components["qxlsx_core"].libs = ["QXlsx"]
+        if Version(self.version) <= "1.4.4":
+            self.cpp_info.components["qxlsx_core"].libs = ["QXlsx"]
+        else:
+            self.cpp_info.components["qxlsx_core"].libs = [f"QXlsxQt{self._qt_version}"]
```

Comment:
maybe wait the next version before introducing branching which is not tested, it may be wrong and trick future contributors when they will package those versions.